src/components/ec-date-range-filter/ec-date-range-filter.vue

```diff
@@ -147,11 +142,30 @@ const numberOfSelectedFilters = computed(() => {
 const isButtonDisabled = computed(() => numberOfSelectedFilters.value <= 0);
 
 function clear() {
-  update(null);
+  update({
+    from: null,
+    to: null,
+  });
 }
+
 function onBlur() {
   emit('blur', { from: fromValueDate.value, to: toValueDate.value });
 }
+
+const isAutoHideEnabled = ref(true);
+const allPopoverOptions = computed(() => ({
+  autoHide: isAutoHideEnabled.value, // autoHide of the ec-filter-popover should be disabled while flatpickr is open, otherwise selecting value in the flatpickr will close this popover too.
```

Comment:
if we want to manage this option regardless `props.popoverOptions.autoHide` value, we need to put this after ..props.popoverOptions